packages/wrangler/src/api/pages/deploy.ts

```diff
@@ -148,6 +148,7 @@ export async function deploy({
 
 	// Grab the bindings from the API, we need these for shims and other such hacky inserts
 	const project = await fetchResult<Project>(
+		undefined,
```

Comment:
Pages is not supported in FedRAMP High, so the next few files relating to pages behaviour just set the `complianceConfig` to undefined, which means they always assume they are in the public region.

Reply:
Now set to `COMPLIANCE_REGION_PUBLIC` constant to make it more self documenting.